morse.rb

```diff
@@ -0,0 +1,58 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z',
+  '.----' => '1',
+  '..---' => '2',
+  '...--' => '3',
+  '....-' => '4',
+  '.....' => '5',
+  '-....' => '6',
+  '--...' => '7',
+  '---..' => '8',
+  '----.' => '9',
+  '-----' => '0',
+  '--..--' => ',',
+  '.-.-.-' => '.',
+  '..--..' => '?',
+  '-..-.' => '/',
+  '-....-' => '-',
+  '-.--.' => '(',
+  '-.--.-' => ')',
+  '.--.-.' => '@',
+  '-...-' => '=',
+  '.-.-.' => '+'
+}.freeze
+
```

Comment:
- [ ] Great work on the morse code decoder however, your code should include three methods (#decode_char, #decode_word, #decode), kindly make sure to implement the `#decode_char` method that translates a morse code symbol into a letter

Reply:
Thank you for your suggestion. We have implemented this.